sources/Afup/Comptabilite/Facture.php

```diff
@@ -476,7 +478,9 @@ function genererFacture($reference, $chemin = null)
             utf8_decode($coordonnees['adresse']) . "\n" .
             utf8_decode($coordonnees['code_postal']) . "\n" .
             utf8_decode($coordonnees['ville']) . "\n" .
-            utf8_decode($pays->obtenirNom($coordonnees['id_pays'])));
+            utf8_decode($pays->obtenirNom($coordonnees['id_pays'])) .
+            $coordonnees['tva_intra'] ? ("\nTVA : ".utf8_decode($coordonnees['tva_intra'])) : null
```

Comment:
En libellé, même si c'est un peu long, j'aurais plutôt vu le libellé complet comme cela :
```suggestion
           ($coordonnees['tva_intra'] ? ("\n" . utf8_decode('N° TVA Intracommunautaire : ' . $coordonnees['tva_intra'])) : null)
```

![Capture d’écran du 2023-12-08 12-42-34](https://github.com/afup/web/assets/320372/97176311-070a-4544-94d1-079d34e880e1)

Reply:
C'est fait, merci